### PR TITLE
feat: move footer to the right hand side

### PR DESCRIPTION
**Pull request by Ruesa18**
References: https://dev.whatwedo.ch/araise/araise-meta/-/issues/29

Move footer to the right side and use same font color as in the footer of the demo websites

### CODE REVIEW

File: src/Resources/views/base.html.twig [hunk 1]
```diff
@@ -163,7 +163,7 @@
                 </div>
 
                 {% block footer %}
-                    <footer class="whatwedo-utility-paragraph text-center mb-2">
+                    <footer class="whatwedo-utility-paragraph !text-neutral-700 text-right mb-2 mx-4">
```
Comment (renestalder) [inline src/Resources/views/base.html.twig:166]: Why does it require the `!important` flag here? Should that text colour overwrite be on the link instead too (with `text-current`) the `!important` flag being removed?

`!important` flags are considered harmful, as difficult to impossible to overwrite.

**Review by renestalder — APPROVED** [review]
@Ruesa18 lgtm, though if you can remove the `!important` flag, consider doing so.

**Comment by renestalder** [comment]
@Ruesa18 Consider linking the issues these changes belong to, not matter they reference internal tickets. I have no way to validate your change with the initial request without any info about later.

**Comment by Ruesa18** [comment]
@renestalder Can do. I just found that since the tickets are internal anyways it's a bit weird. But I will link them in the future 👍

File: src/Resources/views/base.html.twig (hunk 1 above)
Comment (Ruesa18) [reply src/Resources/views/base.html.twig:166]: @renestalder I actually think this is not needed. Don't know why this was added in the demo sites 🤔 
I'll remove it 👍

**Review by tuxes3 — APPROVED** [review]

**Pull request merged**